### PR TITLE
Add documentation on how to connect the focus to Jitsi Videobridge for pubsub statistics and lastN/Dominant speaker switching

### PR DESCRIPTION
**Pull request by davidertel**
Define xmpp message flow for connecting a focus to the Jitsi Videobridge to consume statistics stanzas.

### CODE REVIEW

File: doc/using_statistics.md [hunk 1]
```diff
@@ -134,22 +136,93 @@ Server: Jetty(9.1.5.v20140505)
 }
 ```
 
+**Connecting The Focus To The Publisher For PubSub Method**
+
+Upon start of the Jitsi Videobridge, it will send the following stanza to the focus
```
Comment (dwilson6) [inline doc/using_statistics.md]: The stanza is sent to the pubsub component, not the focus.  Great docs though.

Comment (davidertel) [inline doc/using_statistics.md]: Thanks, trying to save others the time I've spent digging for answers. What do you mean by component here? In our setup, the focus scales our Videobridge cluster by monitoring the pubsub stats messages so we want the Videobridge to publish to the focus.

Comment (dwilson6) [inline doc/using_statistics.md]: The videobridge creates a pubsub node on the pubsub component and then publishes updates to that node. Then the focus subscribes to that pubsub node. The stats are not sent directly to the focus

Comment (davidertel) [inline doc/using_statistics.md]: Ah, our component and focus are kind of jumbled together. I'll update the wording to be more clear. Thank you for the feedback.

**Comment by davidertel** [comment]
@bgrozev thank you for the feedback, would it make sense to change it to "Third, the client must implement the client-to-mixer audio level RTP header extension (RFC6464) and the focus must forward it to the video bridge"

**Comment by bgrozev** [comment]
LGTM

**Comment by davidertel** [comment]
Are there any further changes you would like to see to this documentation?

**Comment by bgrozev** [comment]
No, it looks good. Thank you!

**Pull request merged**